### DISCUSSION TITLE
Is there any room to entertain degrowth in software?

### DISCUSSION THREAD

**Discussion by NAMRATA-WOKE** [Green-Software-Foundation/discussions#30] [Ask Anything]
Came across this [interesting article in the Newyorker](https://www.newyorker.com/news/daily-comment/to-save-the-planet-should-we-really-be-moving-slower). Personally, I am a proponent of degrowth and believe we need to be very intentional in slowing down and course-correcting. Has "degrowth" for a greener future entered the sustainable software discourse? 

Short excerpt:

Isn’t the only way “to mitigate the effects of human activity” to invest in green technologies that actually take us beyond the world of fossil fuels? Shouldn’t we make an all-out push for electric vehicles, heat pumps, and cooktops, not to mention solar panels and wind turbines to supply the necessary electricity? **The [degrowth movement](https://www.newyorker.com/magazine/2020/02/10/can-we-have-prosperity-without-growth)’s answer is, at the least, a muted no.** A green-energy boom, the Canadian journalist Andrew Nikiforuk [wrote](https://www.resilience.org/stories/2023-05-11/i-warned-against-the-green-energy-boom-it-sparked-debate/), would come with “monstrous ecological costs,” because of the mining for the minerals needed to produce and use electricity at the required scale. He cited the energy ecologist Vaclav Smil, who recommends that we return “to living standards of the 1960s” so that we can “consume less, travel less, build less, eat less wastefully.” It’s a view with power: those opposed to new lithium mines or transmission corridors or solar farms are increasingly basing some of their argument on the idea that we should consume less. “If we are to avoid ecological collapse,” the journalist Christopher Ketcham maintains, we must pursue “contraction and simplification, a downsizing of the economy and population, so that Homo sapiens can prosper within the regenerative and assimilative capacity of the biosphere. In other words, we must live within our planet’s biophysical limits.”

**Comment by mrchrisadams** [comment]
I'm not sure I'd call it degrowth, but I think there is.

First of all, I'd suggest that it's worth hearing Jason Hinkel, what one of the highest profile names associated with degrowth says in a recent article:

> I want to address a problem that seems to arise repeatedly in public discussions about green growth and degrowth. Some prominent commentators seem to assume that the debate here is primarily about the question of technology, with green growth promoting technological solutions to the ecological crisis while degrowth promotes only economic and social solutions (and in the most egregious misrepresentations is cast as “anti-technology”).
>
> This narrative is inaccurate, and even a cursory review of the literature is enough to make this clear. In fact, degrowth scholarship embraces technological change and efficiency improvements, to the extent (crucially) that these are empirically feasible, ecologically coherent, and socially just. But it also recognizes that this alone will not be enough: economic and social transformations are also necessary, including a transition out of capitalism. The debate is therefore not primarily about technology, but about science, justice, and the structure of the economic system.

Source: [Monthly Review | On Technology and Degrowth](https://monthlyreview.org/2023/07/01/on-technology-and-degrowth/) by @monthly\_review


We've been funding research specifically in this field for the last year or so, via our [fellowship programme](https://www.thegreenwebfoundation.org/fellowships/), and I'll share a few links to help.

One of last year's fellows, Hannah Smith looked explicitly at applying Donut Economics to her work in the Wordpress Community:

https://www.thegreenwebfoundation.org/fellowships/hannah-smith/

Some of this work is being continued by a fellow from this year, Nahai Badiola, and in the wordpress community specifically in their new sustainability team:
https://www.thegreenwebfoundation.org/news/sustainability-as-an-umbrella-term/

There is lot of other work by our other fellows too

We also maintain a curated library of papers, blog posts, videos and so on at this intersection, in the form of the Green Web Library.
https://www.thegreenwebfoundation.org/news/the-green-web-library-a-collection-of-resources-about-a-just-and-sustainable-internet/

Our focus isn't exclusively on software, but as this post outlines, if you only talk about software, you miss a lot of important context - another one from our fellows:
https://www.thegreenwebfoundation.org/news/what-people-think-building-a-sustainable-internet-involves-vs-what-it-might-actually-be/

Hope that helps.

**Reply by NAMRATA-WOKE** [reply]
@mrchrisadams These resources are so helpful, thank you! I am strongly aligned to Jason's views here: "economic and social transformations are also necessary, **including a transition out of capitalism**. The debate is therefore not primarily about technology, but about science, justice, and the structure of the economic system." 

I am so fascinated by this topic and because it speaks to culture and the kind of mindset we need to cultivate to meet the demands of ecological sustainability in the future, I'd love for the GSF to address it / draw some attention to this brewing research/analysis.
